pysatNASA/instruments/templates/template_cdaweb_instrument.py

```diff
@@ -1,8 +1,20 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
 """Template for a pysat.Instrument support file that utilizes CDAWeb methods.
 
 Copy and modify this file as needed when adding a new Instrument to pysat.
 
+DO NOT include the NRL distribution statement in your new file. Statements for
+new files will have to be approved by NRL separately before the versioned
+release.
```

Comment:
```suggestion
DO NOT include the NRL distribution statement in your new file. Contributions by
NRL developers will need to be cleared separately.
```

Reply:
@rstoneback, is this sufficient? It is difficult to suggest alternatives when there is no feedback.

Reply:
Apologies for the delay.
```suggestion
DO NOT include the NRL distribution statement in your new file. Contributions
by NRL developers will need to be cleared for classified or controlled information 
separately from the pysat pull request process.
```